tests/ExporterTest.php

```diff
@@ -52,6 +52,13 @@ public function exportProvider()
             'export int 1'                  => [1, '1'],
             'export float 1.0'              => [1.0, '1.0'],
             'export float 1.2'              => [1.2, '1.2'],
+            'export float 1 / 3'            => [1 / 3, '0.3333333333333333'],
+            'export float 1 - 2 / 3'        => [1 - 2 / 3, '0.33333333333333337'],
```

Comment:
previously `0.3333333333333333` was dumped (as for `1 / 3`, but the stored value is slighly different)